firefox-plugins-installer

```diff
@@ -43,11 +43,11 @@ JAVA_LINK=${PLUGINS_DIR}/libnpjp2.so
 # previous versions are often removed from the server, so it is safest
 # for an automated script to use the stable version in the archive
 # http://fpdownload.macromedia.com/get/flashplayer/pdc/11.2.202.643/install_flash_player_11_linux.x86_64.tar.gz
-FLASH_VERSION=11.2.202.644
-FLASH_SHA256SUM=aba0e2f5940fc2588357e93f7bad84a0db51792b2f2bd1d24be9ef491e87c041
-FLASH_FILENAME=fp_${FLASH_VERSION}_archive.zip
+FLASH_VERSION=24.0.0.221
```

Comment:
Good point.  Since we would need the sha256sum, I think the second approach is the way to go.  We just need to be careful to make the change in such a way that we don't break existing users of the chrome plugin updater.  I assume as long as we keep the existing "URL" and "SHA256SUM" variable names for PPAPI and just add new strings for NPAPI, we should be fine, correct?